test/dlc_tests/container_tests/bin/efa/testEFA

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+set -ex
+
+hosts_file=$1
+openmpi_path_for_efa=$2
+hosts=2
+
+gpu_count=$(nvidia-smi -L | wc -l)
+nodes=$(($gpu_count * $hosts))
+
+PRETTY_NAME=$(cat /etc/os-release | grep PRETTY_NAME)
+TRAINING_LOG="/test/logs/testEFA.log"
+
+#if [[ $PRETTY_NAME == *"Amazon Linux AMI 2018.03"* ]]; then
+#    CUDA_VERSIONS=('10.0' '10.1' '10.2')
+#else
+#    CUDA_VERSIONS=('11.3')
+#fi
+
+if [[ $PRETTY_NAME == *"Ubuntu"* ]]; then
+    open_mpi_lib_path=/opt/amazon/openmpi/lib
+else
+    open_mpi_lib_path=/opt/amazon/openmpi/lib64
+fi
+
+check_ring_single_node(){
+    echo "Running ring"
+    $openmpi_path_for_efa -n 3 --host localhost --oversubscribe \
+        -x RDMAV_FORK_SAFE=1 -x FI_EFA_USE_DEVICE_RDMA=1 -x NCCL_PROTO=simple --mca pml ^cm \
+        -x LD_LIBRARY_PATH=/usr/local/cuda/efa/lib:/usr/local/cuda/lib:/usr/local/cuda/lib64:/usr/local/cuda:$LD_LIBRARY_PATH \
```

Comment:
can you check if there is dir `/usr/local/efa` as I see DLC install ofi plugin at `/usr/local` https://github.com/aws/deep-learning-containers/blob/master/pytorch/training/docker/1.13/py3/cu117/Dockerfile.gpu#LL292C35-L292C45

Reply:
No, the EFA libs come from `/opt/amazon/efa/lib`. There doesn't seem to be a `/usr/local/efa`.